src/uu/more/src/more.rs

```diff
@@ -179,6 +185,22 @@ pub fn uu_app() -> Command {
                 .help("Squeeze multiple blank lines into one")
                 .action(ArgAction::SetTrue),
         )
+        .arg(
+            Arg::new(options::PLAIN)
```

Comment:
if it does nothing, please remove .help 
and add the clap option to make it not listed :)
thanks